docs/releases/v1.4.2.md

```diff
@@ -0,0 +1,30 @@
+# Release notes
+
+
+## Changelog
+
+Changes between `1.4.0` and this release: `1.4.1`
+
+- Refactor `dual-nginx` package to use the base of the `nginx` *(simple)* deployment.
+
+### Upgrade path
+
+If you already have deployed the `dual-nginx` package you should follow this procedure:
```

Comment:
Just double checking here. There's no change for the single controller?

Reply:
No. We ensured no change was made in the single nginx deployment. It is now the base of the dual nginx ingress controller deployment and that's why dual-nginx ingress deployment needs special upgrade procedure ;)

Reply:
great! Thanks